docs/internals/security.txt

```diff
@@ -91,15 +99,9 @@ email message, signed with the Django release key, containing:
 * The date on which the Django team will apply these patches, issue
   new releases and publicly disclose the issue.
 
-Simultaneously, the reporter of the issue will receive notification of
-the date on which we plan to take the issue public.
-
 On the day of disclosure, we will take the following steps:
 
-1. Apply the relevant patch(es) to Django's codebase. The commit
-   messages for these patches will indicate that they are for security
-   issues, but will not describe the issue in any detail; instead,
-   they will warn of upcoming disclosure.
+1. Apply the relevant patch(es) to Django's codebase.
```

Comment:
It's inaccurate based on current practices. Patches now also include release notes.